### DISCUSSION TITLE
River with Colab

### DISCUSSION THREAD

**Discussion by ercangunbilek** [online-ml/river#998] [Q&A] [answered]
Latest version of river doesn't work on Google Colab. I tried `pip install river==0.11.1` but it didn't work. How can I use latest version of river on Colab ?

**Comment by kulbachcedric** [comment, marked as answer]
Hi @ercangunbilek,

river only works with version 0.10.1 on Colab.

Best 
Cedric

**Reply by MaxHalford** [reply]
Do you know the reason why? Is there a traceback we can inspect?

**Reply by kulbachcedric** [reply]
`!pip install river==0.11.1`

>  Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/
> ERROR: Could not find a version that satisfies the requirement river==0.11.1 (from versions: 0.7.0, 0.7.1, 0.8.0, 0.9.0, 0.10.0, 0.10.1)
> ERROR: No matching distribution found for river==0.11.1

Colab runs on Python 3.7.13

Adding `cp37-*` to the PyPi workflow could do the job, right?

**Reply by MaxHalford** [reply]
Mmm I didn't realize Colab only had support for Python 3.7. I guess we can add Python 3.7 into the next release.
